pgmq-extension/sql/pgmq.sql

```diff
@@ -734,12 +734,12 @@ BEGIN
     qtable, partition_col
   );
 
-  IF NOT pgmq._belongs_to_pgmq(FORMAT('%I', qtable)) THEN
+  IF NOT pgmq._belongs_to_pgmq(qtable) THEN
       EXECUTE FORMAT('ALTER EXTENSION pgmq ADD TABLE pgmq.%I', qtable);
   END IF;
 
   PERFORM public.create_parent(
-    'pgmq.' || quote_ident(qtable),
+    FORMAT('pgmq.%s', qtable),
```

Comment:
Might be worth adding a comment here that the pg_partman docs say:

> `p_parent_table` - the existing parent table. MUST be schema qualified, even if in public schema.

Reply:
done!